Moved CSS styles to red-inside

theilman7 opened the pull request:
Please reference issue: Move agency contact module styles from agency-homepage.css to red-inside.css #275

- Removed styles for the contact sidebar box, contained in agency-homepage.min.css.
- Moved styles mentioned above to red-inside.css.

theilman7 committed:
Moved CSS styles to red-inside

Please reference issue: Move agency contact module styles from agency-homepage.css to red-inside.css #275

- Removed styles for the contact box, contained in agency-homepage.min.css.
- Moved styled mentioned above to red-inside.css.

martypowell commented:
@danfox01 @theilman7 we actaully do not want to update red-inside, that is an existing style sheet that kinda lived on from pre github days. This should be integrated into the inside-responsive template i believe. 

It would probably be worth creating a new file for this stuff, as it seems like it's pretty focused, and then include it with the others. If you have any questions, please let me know.

danfox01 commented:
@martypowell thanks the info on this. @theilman7 as long as you are comfortable, please go ahead with Marty's recommendation.

theilman7 commented:
@martypowell @danfox01 Thanks Marty! I can work on doing that. Going to follow up with Marty to see if there is a preferred file under the /partials directory you would prefer these styles added to. I see inside-responsive.scss `@import`s quite a bit from there, so I want to be sure I place it in the desired file. Alternatively, I can create a file (maybe sidebar or enhancements?) and `@import` it in to inside-responsive.scss.

Thanks again all!

theilman7 committed:
Revised Code for Pull Request

Revising as per feedback from Pull Request #276 . Removed styles from red-inside. Created a new stylesheet, titled sideBar.scss. Added styles to new stylesheet and included the file to inside-responsive.

theilman7 commented:
Revised and pushed new commit based upon feedback. Let me know when you have a chance to review and if you would like to see anything revised.
 @martypowell @danfox01

theilman7 commented:
@danfox01 @martypowell Just wanted to ping a friendly reminder on this thread regarding the updated pull request.

Thanks all!
_TH

martypowell commented:
@theilman7 this looks good, has this been uploaded to dev?
